src/app/providers.tsx

```diff
@@ -4,13 +4,19 @@ import {ViewerProvider} from 'context/viewer-context'
 import {CioProvider} from 'hooks/use-cio'
 import {ThemeProvider} from 'next-themes'
 import TrpcProvider from 'app/_trpc/Provider'
+import PostHogClient from 'lib/posthog-client'
+import {PostHogProvider} from 'posthog-js/react'
 
 export function Providers({children}: {children: React.ReactNode}) {
+  const posthog = PostHogClient
+
   return (
     <ThemeProvider attribute="class">
       <ViewerProvider>
         <TrpcProvider>
-          <CioProvider>{children}</CioProvider>
+          <CioProvider>
+            <PostHogProvider client={posthog}>{children}</PostHogProvider>
+          </CioProvider>
```

Comment:
We shouldn't need to import PostHogClient in the other files if we are wrapping the tree in a provider

```
import { usePostHog } from 'posthog-js/react'
```

`usePostHog` is the hook that would provide posthog to us